url-updater/src/main/java/com/devonfw/tools/ide/url/updater/gcloud/GCloudUrlUpdater.java

```diff
@@ -0,0 +1,46 @@
+package com.devonfw.tools.ide.url.updater.gcloud;
+
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.WebsiteUrlUpdater;
+
+import java.util.regex.Pattern;
+
+public class GCloudUrlUpdater extends WebsiteUrlUpdater {
+
+  @Override
+  protected String getTool() {
+
+    return "gcloud";
+  }
+
+  @Override
+  protected void addVersion(UrlVersion urlVersion) {
+
+    String baseUrl = "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${version}-${os}-";
+
+    doAddVersion(urlVersion, baseUrl + "x86_64.zip", WINDOWS);
+    //TODO add Linux and MacOS
```

Comment:
We do not provide `gcloud` for Linux and MacOS. The reason for that is that we do not provide python for these OS's and that is a requirement for `gcloud.` Should we still add these with the hope of fixing Python on these?

Reply:
Yes. If a version is added to ide-urls with only windows and we later change the code to also create linux and mac URLs, those will not be added without deleting the old versions. So we should already add all OS from the start.

Reply:
Done